lib/rules/sort-keys.js

```diff
@@ -159,10 +167,45 @@ module.exports = {
                 const numKeys = stack.numKeys;
                 const thisName = getPropertyName(node);
 
+                // Get tokens between current node and previous node
+                const tokens = stack.prevNode && sourceCode
+                    .getFirstTokensBetween(stack.prevNode, node, { skip: 0, includeComments: true });
```

Comment:
Can we use `getTokensBetween`? `getFirstTokensBetween` without `count` looks confusing.

```suggestion
                    .getTokensBetween(stack.prevNode, node, { includeComments: true });
```

Reply:
Updated 👍🏻